utils/zerovec/src/lib.rs

```diff
@@ -78,25 +63,136 @@
 //!     #[serde(borrow)]
 //!     nums: ZeroVec<'data, u32>,
 //!     #[serde(borrow)]
+//!     chars: ZeroVec<'data, char>,
+//!     #[serde(borrow)]
 //!     strs: VarZeroVec<'data, str>,
 //! }
 //!
 //! let data = DataStruct {
 //!     nums: ZeroVec::from_slice(&[211, 281, 421, 461]),
+//!     chars: ZeroVec::from_slice(&['ö', '冇', 'म']),
 //!     strs: VarZeroVec::from(&["hello", "world"]),
 //! };
 //! let bincode_bytes = bincode::serialize(&data)
 //!     .expect("Serialization should be successful");
-//! assert_eq!(54, bincode_bytes.len());
+//! assert_eq!(74, bincode_bytes.len());
```

Comment:
Backwards this reads. You use `assert_eq!(actual, expected)` almost everywhere.

Also a few instances below.